frontend/src/views/Landing.tsx

```diff
@@ -1,27 +1,205 @@
-import React from 'react';
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import copy from 'copy-to-clipboard';
 import { PrimaryButtonLink, TextLink } from '../components/core/Link';
-import { MainHeaderText, LandingHeaderTitle } from '../components/core/Text';
+import { Image, ShadowImage } from '../components/core/Image';
+import {
+  MainHeaderText, LandingHeaderTitle, SecondaryHeaderText, LargeText,
+} from '../components/core/Text';
+import { ColumnContainer, RowContainer, Separator } from '../components/core/Container';
+import { CopyIndicator, CopyIndicatorContainer, InheritedCopyIcon } from '../components/special/CopyIndicator';
+import { InheritedTextButton } from '../components/core/Button';
+
+const CreateAccountButtonLink = styled(PrimaryButtonLink)`
+  margin: 10px 0;
+`;
+
+const HeroTextContainer = styled.div`
+  text-align: left;
+  
+  @media(max-width: 1000px) {
+    text-align: center;
+    margin-bottom: 40px;
+  }
+`;
+
+const HeroText = styled(LandingHeaderTitle)`
+  font-size: ${({ theme }) => theme.fontSize.xxxLarge};
+  margin: 5px 0;
+  line-height: 1.2;
+  
+  @media(max-width: 1450px) {
+    font-size: ${({ theme }) => theme.fontSize.xxLarge};
+  }
+`;
+
+const HeroSubtitleText = styled(MainHeaderText)`
+  @media(max-width: 1450px) {
+    font-size: ${({ theme }) => theme.fontSize.mediumLarge};
+  }
+`;
+
+const BackgroundCircleRow = styled(RowContainer)`
+  background-image: url("/landing/background_circles.png");
+  background-repeat: no-repeat;
+  background-position: center;
+  padding: 150px 30px;
+  margin-bottom: 50px;
+  
+  @media(max-width: 1000px) {
+    padding: 0;
+    background-image: none;
+  }
+`;
+
+const CalToActionColumn = styled(ColumnContainer)`
+  flex: 0 0 60%;
+
+  @media(max-width: 1000px) {
+    width: 70%;
+    margin: 0 auto;
+  }
+`;
 
 function LandingPage() {
```

Comment:
How about something like this? I moved the circles more to the outside and made it fixed height, so it doesn't overlap (at least on common screen sizes, not sure about heavily zoomed in/out views)

<img width="1347" alt="Circles moved to the borders" src="https://user-images.githubusercontent.com/24768574/120586344-37260680-c3e8-11eb-9ff6-f07c2aeb46a9.png">

Imo I like the color that it adds, but feel free to lmk what you think (this also affects the display on the create and join pages)

Reply:
I think you're right actually, that looks great! It changes the setup on the create and join room pages as well, but I think the new circle placement for those looks good too, so no problems there.

My only nit would be to move the pink circle just very slightly right to prevent it from going off the screen, in at least most cases. This is just a tiny change / nit, it shouldn't need to move much at all 👍 

Thanks for updating that!

Reply:
Sg, I'll leave it as is, since I realized people will only see this page if logged out, which has both a Register and Login link which will overflow no matter what lol

Reply:
Whoops, that was my response to the below thing about the header

As for this, sounds good, I'll move the pink circle slightly to the left (I think that's what you meant in order to not make it go off-screen right? Lmk if not)